src/controllers/dlgprefcontrollerdlg.ui

```diff
@@ -53,6 +53,425 @@
         </widget>
        </item>
        <item row="10" column="0" colspan="2">
+        <layout class="QGridLayout" name="gridLayout_8" rowstretch="0">
+         <item row="0" column="0">
+          <widget class="QGroupBox" name="groupBoxDeviceInfo">
+           <property name="sizePolicy">
+            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+             <horstretch>0</horstretch>
```

Comment:
are these device infos visible all the time? Are the relevant to all users all the time? Would it make sense to hide it behind `--developer` or `--controller-debug`?

Reply:
--controllerDebug and later an always visible, collapsible region?

Reply:
This informaton is also intended for the end-user! An end-user needs it to identify the correct device out of many with similar names. It should be visible for the end-user by default therefore.
But once the end-user identified the correct device, and the mapping is successful loaded and working, he no longer needs these details.

Reply:
right, but how user friendly is that? In the long-term we probably need a different mechanism for identifying devices...

Reply:
Yes, the long term target is Plug&Play. That is why I make the device data accessible in this first step.

Reply:
nice. So leave like this for now and `--developer` once plug'n'play is achieved?